components/profile_popover/profile_popover.jsx

```diff
@@ -200,6 +200,7 @@ class ProfilePopover extends React.Component {
         delete popoverProps.hasMention;
         delete popoverProps.dispatch;
         delete popoverProps.enableWebrtc;
+        delete popoverProps.showEmailAddress;
```

Comment:
Injected by the connector but wasn't removed, like `enableWebrtc`, and ended up incorrectly as a DOM property.

Reply:
I think we are not using showEmailAddress at all, so, the solution would be remove that from the connect function.

Reply:
Ooh, I like that better -- somehow, I thought this was being passed in via a plugin, but you're right that it's only part of the pluggable unit tests.